apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -85,4 +92,37 @@ export class SubscribersController {
       })
     );
   }
+
+  @Get('/preferences')
+  @ExternalApiAccessible()
+  @UseGuards(JwtAuthGuard)
+  async getSubscriberPreference(@UserSession() user: IJwtPayload) {
+    const command = GetSubscriberPreferenceCommand.create({
+      organizationId: user.organizationId,
+      subscriberId: user._id,
+      environmentId: user.environmentId,
+    });
+
+    return await this.getSubscriberPreferenceUsecase.execute(command);
+  }
+
+  @Patch('/preference/:templateId')
```

Comment:
I would suggest doing the following rest: `/subscribers/:subscriberId/preferences/:templateId`